### DISCUSSION TITLE
Regarding enbling monitor mode in Samsung S20 phone

### DISCUSSION THREAD

**Discussion by Prateek5826** [seemoo-lab/nexmon_csi#375] [General]
Hi, I need to put my samsung s20 phone into monitor mode, I searched the firmware patches but could not find it, can you please suggest me the steps and also the firmware. (BCM4375b1)

**Comment by zeroby0** [comment]
https://github.com/seemoo-lab/nexmon/issues/519

https://github.com/seemoo-lab/nexmon/issues?q=s20
